shared/js/toc.jsx

```diff
@@ -52,12 +52,14 @@ loop.shared.toc = (function(mozL10n) {
       });
     },
 
-    // XXX akita: add jsdoc
-    addTile: function(url) {
+    /**
+     * Adds a new tile to the ToC
+     */
+    addTile: function(metadata) {
       var tiles = this.state.tiles;
       tiles.push({
-        location: url,
-        description: url
+        location: metadata.url,
+        description: metadata.description
```

Comment:
@fcampo It's another bug -> https://bugzilla.mozilla.org/show_bug.cgi?id=1273506